pubsub/google/cloud/pubsub_v1/types.py

```diff
@@ -104,6 +104,39 @@
     )
 
 
+if sys.version_info >= (3, 5):
```

Comment:
I filled in the changes in the generator and our templates, but this looks like something else. Please take a look @anguillanneuf @plamut.

Reply:
This was added by me, as injecting custom docs to namedtuple members is only possible in Python 3.5+.

Or were you maybe referring to something else?

Reply:
Nope, just wanted to make sure this was expected. Thanks!